install.sh

```diff
@@ -330,9 +330,12 @@ nvm_do_install() {
   BASH_OR_ZSH=false
 
   if [ -z "${NVM_PROFILE-}" ] ; then
-    echo "=> Profile not found. Tried ${NVM_PROFILE} (as defined in \$PROFILE), ~/.bashrc, ~/.bash_profile, ~/.zshrc, and ~/.profile."
+    local TRIED_PROFILE
+    if [ -n "${PROFILE}" ]; then
+      TRIED_PROFILE="${NVM_PROFILE} (as defined in \$PROFILE),"
+    fi
+    echo "=> Profile not found. Tried ${TRIED_PROFILE-} ~/.bashrc, ~/.bash_profile, ~/.zshrc, and ~/.profile."
```

Comment:
When `TRIED_PROFILE` is empty, this will be `Tried  ~/.bashrc` - note the double space. iow, I think the space should go on line 335, and here, there should be no space after `${TRIED_PROFILE-}`.

Reply:
hmmmm ... Just thought about the code readability so I did this, I can tweak it.